src/conic_algorithm.jl

```diff
@@ -2051,7 +2060,7 @@ function add_sep_cut_exp!(m, add_cuts::Bool, r, s, t)
 
             # K* separation cut on (r,s,t) is (t/r, -2*log(exp(1)*t/2r), -2)
             u_val = t_val/r_val
-            v_val = -2.*(1. + log(u_val/2.))
+            v_val = -2 .* (1. + log(u_val/2.))
```

Comment:
here and below the "." was supposed to make 2 a float, not to broadcast the *
```suggestion
            v_val = -2.0 * (1.0 + log(u_val/2.0))
```